build/codebase-check.sh

```diff
@@ -192,7 +230,7 @@ cleanup
 SUMMARY_FILE="${ARTIFACT_DIR}/summary-${ERROR_FILE_NAME}"
 
 echo ""
-echo "****" | tee -a ${SUMMARY_FILE}
```

Comment:
why `-a` removed?

Reply:
It's the first instance of `tee`, so if you want to re-run it locally, it should be overwriting the file, not appending to it.